src/inflators/model.tsx

```diff
@@ -139,6 +153,16 @@ export function inflateModel(world: HubsWorld, rootEid: number, { model }: Model
     }
   });
 
+  // Hubs loop-animation component is defined at glTF node level.
```

Comment:
Not sure I would have bothered trying to consolidate these at this stage. Writing an inflator for the GLTF data that added a LoopAnimation component, who's data is an array of clips, and then having a system that find's the ancestor mixer to play those clips on could have worked. I suppose this gets us closer to where we would eventually like to be with it but adding more special cases to `ignoredComponents` is definitely something we should really try and minimize. Animations in three are tricky though and quite tied to GLTF loading and the threejs hierarchy, so I can see why it might be justified.